estruturacao.py

```diff
@@ -0,0 +1,40 @@
+import sys
+
+
+def lerArquivo(arquivo):
+	arq = open(arquivo, 'r')
+	test = arq.readlines()
+	test = [ x.replace("\n", "").split(" ") for x in test]
+	del test[0]
+	return test
+
+def converteCoord(linhas, lista):
+	test1 = []
+	for y in range(totalLinhas):
+		test1.append([float(x) for x in lista[y]])
```

Comment:
Mano, tu consegue ver que estamos iterando duas vezes nesses arrays gigantes??
O que acha de fazer uma função que recebe um array de strings e já converte os primeiros valores para float e o ultimo para int? Pois ai não precisaríamos fazer esse for.
algo como: 
```suggestion
	lambda x: [float(x) if arr.index(x) < len(arr) - 1 else int(x) for x in arr]
```
sem a anotação de lambda, e sim como uma função normal.

Reply:
Tem até uma forma melhor do que essa! Sem ficar calculando o `len(arr)` a cada iteração.